src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js

```diff
@@ -47,6 +53,9 @@ class ExchangeWidget extends Component {
       }
 
       this.walletInputEl.focus();
+      this.destinationTagInputEl.focus();
+      this.paymentIdInputEl.focus();
+      this.memoInputEl.focus();
```

Comment:
Calling `focus` on several elements will place the cursor at the last element (field) the function was called on.